modules/flowable-rest/src/main/java/org/flowable/rest/service/api/runtime/task/TaskResource.java

```diff
@@ -51,17 +49,20 @@
 import io.swagger.annotations.ApiResponses;
 import io.swagger.annotations.Authorization;
 
+import java.util.HashMap;
+import java.util.Map;
```

Comment:
Project standards are to include all java packages first as was originally done in this file.

Reply:
Ok, solved.